src/test/java/seedu/address/logic/commands/EditStudentCommandTest.java

```diff
@@ -36,21 +36,6 @@
 
     private Model model = new ModelManager(getTypicalAddressBook(), new UserPrefs());
 
```

Comment:
Is there any reason why you remove this test?
Cause i think still need to check if the other fields can be edited